plugin/arcconf/arcconf.py

```diff
@@ -439,9 +440,13 @@ def _arcconf_ld_to_lsm_vol(arcconf_ld,
             admin_status = Volume.ADMIN_STATE_DISABLED
         else:
             admin_status = Volume.ADMIN_STATE_ENABLED
-        plugin_data = "%s:%s:%s" % (ctrl_num, array_num, ld_num)
-
-        volume_id = array_num
+        # plugin_data = "%s:%s:%s:%s" % (ctrl_num, array_id, ld_id, raid_level)
+        stripe_size = arcconf_ld['StripeSize']
+        full_stripe_size = arcconf_ld['fullStripeSize']
+        ld_state = arcconf_ld['state']
+        plugin_data = {'ctrl_id': ctrl_num, 'array_id': array_id, 'ld_id': ld_id, 'ld_state': ld_state,
```

Comment:
`plugin_data` should be `string`

Reply:
Done - commit id - a70a0a1a4fffd20be4239def1f0e0a2d91d1b41b